src/styles/components/_form.sass

```diff
@@ -26,6 +26,8 @@ form
   .text-field-hint
     font-size: 12px
     color: #ccc
+    .error
+      color: #771d1d
 
```

Comment:
To me the contrast is not fine. I would select another color tone which highlights from the background

Reply:
I settled for the darker blue in the modal